package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Assists in the creation of ATNA audit trail messages for IHE profiles.",
   "main": "index.js",
   "scripts": {
-    "test": "tap --cov test.js"
+    "pretest": "npm run gen:rng",
+    "test": "tap --cov test.js",
+    "gen:rng": "sed 's/#.*//' dicom.rnc > temp.rnc && trang temp.rnc dicom.rng && rm temp.rnc"
```

Comment:
I didn't want to take out the comments as I've found them pretty useful to refer back to when adding more of these audit messages. Also, since the .rng is just a intermediary file in this case, I didn't want to have it checked into the repo either. I agree it's not great for non-unix environments, but not sure how to do this better while keeping it automatic.